src/app/(auth)/sign-in/page.tsx

```
@@ -1,11 +1,21 @@
 import { SignInForm } from '../_components/SignInForm'
+import { auth } from '@/lib/auth';
+import { redirect } from 'next/navigation';
+import { headers } from 'next/headers';
 
-const  SignIn = () => {
+const SignIn = async () => {
+    const session = await auth.api.getSession({
+        headers: await headers()
+    });
+
+    if (!!session) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)